cluster/config-defaults.yaml

```diff
@@ -288,6 +288,21 @@ skipper_ingress_backend_traffic_algorithm: "traffic-segment-predicate"
 # TODO: after a while we can remove this and hardcode (2023-06-30)
 skipper_ingress_default_lb_algorithm: "powerOfRandomNChoices"
 
+# Open Policy Agent
```

Comment:
Let's improve the comment of this config item. Currently it's not clear what it does.